pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -225,25 +234,22 @@ func (p Provisioner) Revoke(ob *bktv1alpha1.ObjectBucket) error {
 	}
 	logger.Infof("Revoke: denying access to bucket %q for OB %q", p.bucketName, ob.Name)
 
-	bucket, _, err := cephObject.GetBucket(p.objectContext, p.bucketName)
+	bucket, err := p.adminOpsClient.GetBucketInfo(context.TODO(), admin.Bucket{Bucket: p.bucketName})
 	if err != nil {
 		logger.Errorf("%v", err)
 	} else {
 		if bucket.Owner == "" {
-			return errors.New("cannot find bucket owner")
+			return errors.Errorf("failed to find bucket %q owner", p.bucketName)
 		}
 
-		user, code, err := cephObject.GetUser(p.objectContext, bucket.Owner)
-		// The user may not exist.  Ignore this in order to ensure the PolicyStatement does not contain the
-		// stale user.
-		if err != nil && code != cephObject.RGWErrorNotFound {
+		user, err := p.adminOpsClient.GetUser(context.TODO(), admin.User{ID: bucket.Owner})
+		if err != nil && !errors.Is(err, admin.ErrNoSuchUser) {
```

Comment:
I fully don't understand this check, is it ignoring `No such user error`?

Reply:
I'll change that